docs/eigenda/operator-guides/networks/goerli.md

```diff
@@ -4,4 +4,4 @@ sidebar_position: 1
 # Goerli (Deprecated)
 
 The Goerli network has been deprecated. If you are interested in launching an
-EigenDA test operator please follow the [Holesky](./holesky.md) testnet guide.
+EigenDA test operator please follow the [Holesky](./holesky.mdx) testnet guide.
```

Comment:
Can you please remove this file modification? Asking b/c it clashes with this change: https://github.com/Layr-Labs/eigenlayer-docs/pull/158/files